app.js

```diff
@@ -341,17 +341,16 @@ const topicExists = function topicExists(id) {
   return docManager.fileExists(fileName, id);
 };
 
-// TODO change to a DELTE /topic/:id
-app.delete('/file', (req, res) => {
+app.delete('/topic/:id', (req, res) => {
```

Comment:
Since students are not even allowed to create topics, we should call 
get /Api/Permissions/Topics/{topicId}/Permission/IsAllowedToEdit before deleting the topic.

We maybe should also not delete the topic, but move it in a backup folder, so we can later restore it.